kfp/transform_workflows/.make.transforms_workflows

```diff
@@ -62,5 +62,5 @@ ${VENV_ACTIVATE}: ${REPOROOT}/.make.versions ${REPOROOT}/kfp/requirements.env ${
 
 .PHONY: .transforms_workflows.clean
 .transforms_workflows.clean:
-	rm -rf *_wf.yaml || true
```

Comment:
why this line is removed?

Reply:
It might be useful to keep it around. Its in gitignore anyways

Reply:
I agree with Boris, it is very confused that the pipeline yamls disappear